From 84430a5f5532ec39902241760b797fd24d26410b Mon Sep 17 00:00:00 2001
From: Wei Wang <wvw@google.com>
Date: Wed, 7 Jun 2017 14:06:47 -0700
Subject: [PATCH 6/7] sanders: Use mke2fs to create ext4 images

Set TARGET_USES_MKE2FS := true to use new EXT4 tool chain

Bug: 62421233
Change-Id: Id6e5c68b2d7854e5330ad87443b974361a4bff9d
---
 BoardConfig.mk | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/BoardConfig.mk b/BoardConfig.mk
index ad18080e..ade5b162 100644
--- a/BoardConfig.mk
+++ b/BoardConfig.mk
@@ -225,6 +225,9 @@ TARGET_LD_SHIM_LIBS := \
 # Thermal
 USE_DEVICE_SPECIFIC_THERMAL := true
 
+# Use mke2fs to create ext4 images
+TARGET_USES_MKE2FS := true
+
 # Wifi
 BOARD_HAS_QCOM_WLAN              := true
 BOARD_HAS_QCOM_WLAN_SDK          := true
-- 
2.18.0

